### ISSUE TITLE
Both sync and scrobbling is no longer working for neither movies nor TV shows

### ISSUE THREAD

**Issue by lthustrup** [trakt/Plex-Trakt-Scrobbler#449]
Both syncing and scrobbling to Trakt has stopped working for my Plex server around Monday June 12, 2017 - that is the last day anything is in the history on my Trakt profile.
It may have been around the time I last updated the Plex server, when it prompted me about a newer version, I haven't checked in to that, not sure how.

My current Plex version when I became aware of this issue was 1.5.7.4016, and I see there is another newer update available.
I just tried restarting Plex Media Server, but syncing still doesn't add anything.
Then I updated to the currently newest version 1.7.5.4035 and tried a sync again.
Now I don't get any errors or warning like before the update, but the history and show status are still not updated on Trakt.

Can you please advise and help me get it running again? Thanks.

Before I tried restarting and updating PMS, I pulled the following from Plex-Trakt plugin log file
It says "ignoring duplicate "add" action.
Here is a sample for a movie:
2017-07-02 21:50:42,410 (18c0) :  DEBUG (logkit:13) - [plugin.sync.push] Processing "The Founder" [('imdb', 'tt4276820')]
2017-07-02 21:50:42,411 (18c0) :  INFO (logkit:16) - [plugin.pts.action_manager] Ignoring duplicate "add" action (history: {'performed': ['add'], 'sent': ['add']})
2017-07-02 21:50:42,411 (18c0) :  DEBUG (logkit:13) - [plugin.sync.push] watch() - Invalid action for "The Founder" [('imdb', 'tt4276820')] (already scrobbled or duplicate action)

I also included samples from syncing of TV shows and scrobbling of a TV show episode in the attached file.

[snippets from Snippets from com.plexapp.plugins.trakttv.log.txt](https://github.com/trakt/Plex-Trakt-Scrobbler/files/1118095/snippets.from.Snippets.from.com.plexapp.plugins.trakttv.log.txt)

**Comment by TravisBickle76** [comment]
Same issue here. Started on June 11th.

**Comment by lthustrup** [comment]
I got it working again, and thought I'd share, what I did:
I found out, that I was running quite an old version of the plugin - 0.9.1.29-master. Not sure why I didn't think of checking that earlier.
So I removed the old version, just to be sure, there wouldn't be any mixup when installing the new version, and then used Unsupported Appstore from the WebTools plugin (which I already had for other reasons) to install the latest version.
Just follow the guide for your OS from the Github page of the plugin: https://github.com/trakt/Plex-Trakt-Scrobbler/wiki/Installation
Now I'm running version 1.3.2 of the Trakt.tv plugin, and the sync seemed to work correctly - now on to test scrobbling ;)
Thanks to the team of developers for this plugin - Great job!
